lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -140,8 +140,17 @@ sub group_overview {
     return $self->reply->not_found unless $group;
 
     my $res = $self->_group_result($group, $limit_builds);
-    my @comments = $group->comments->all;
-    for my $comment (@comments) {
+    my @comments;
+    my @pinned_comments;
+    for my $comment ($group->comments->all) {
+        # find pinned comments
+        if ($comment->user->is_operator && CORE::index($comment->text, 'pinned-description') >= 0) {
```

Comment:
To restrict pinning comments. Only admins and operators are allowed to do it. If anybody should be allowed, then the check should be removed of course.

Hiding the keyword would be nice indeed. Could this be done when rendering the markdown?